test/ASRock-Single-Ryzen.json

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
this is more of a nit, but Go convention is to place such files to `smbios/testdata/*`. 

And reference them like `testdata/foo` from the unit-test (unit-test always has working directory set to the package directory)

Reply:
Moved them, thanks for the tip! I was unsure about this, since I'm originally a Java developer and I didn't know what the convention is. So I learned something new in the process 😄 

I also fixed some linter errors.